woocommerce/woocommerce-gateway-stripe
# chore: remove has-icon-border class

# Changes proposed in this Pull Request:

I noticed we're applying an `has-icon-border` class name.
When applied, the class name targets a child `svg` element to apply a border.

The thing is, we don't have any child `svg` elements.
![Screen Shot 2021-10-05 at 3 43 59 PM](https://user-images.githubusercontent.com/273592/136099808-1686a3c5-beda-4b7c-b08a-fe86eba2ac0b.png)


It looks like this was ported over from WCPay.
We don't need it.

# Testing instructions
If you want to test this out:
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc_stripe-onboarding_wizard
- There is no change in behavior with the icons

Before:
![Screen Shot 2021-10-05 at 3 46 54 PM](https://user-images.githubusercontent.com/273592/136100249-6983e03f-797f-476e-99a9-406dad1b5a8e.png)
![Screen Shot 2021-10-05 at 3 47 10 PM](https://user-images.githubusercontent.com/273592/136100235-c78eaf93-b2a3-47ac-9c97-7cdbfc3407a0.png)


After:
![Screen Shot 2021-10-05 at 3 46 01 PM](https://user-images.githubusercontent.com/273592/136100096-f7eaffcd-41f5-44e7-9f82-b459cd1df02c.png)
![Screen Shot 2021-10-05 at 3 45 49 PM](https://user-images.githubusercontent.com/273592/136100062-c2ca37ec-f92f-495e-ac2d-cdee24d0a3d5.png)
